### DISCUSSION TITLE
Reports, Logging, and Statistics

### DISCUSSION THREAD

**Discussion by mengelsen** [cupcakearmy/cryptgeon#115] [Ideas]
Would it be worth adding "reports" to the roadmap?

Just thinking that it would be nice to know how many notes are created on any given day. How many notes are still unread (with how many views remaining).

Cryptgeon reports/logs could be written to a file, accessed via a password protected WebGUI, or printed on-screen via a Terminal command.

**Comment by cupcakearmy** [comment]
I'm also curious about usage, however it goes a bit agains the core philosophy.
Maybe an opt in option would be interesting. It could also display it on the web ui "x notes shared in total, y today" or something.
